### DISCUSSION TITLE
Extend the index

### DISCUSSION THREAD

**Discussion by thomasmach** [EricDarve/numerical_linear_algebra#239] [Ideas]
I just tried to find the _power iteration_ in the book. Unfortunately, it is not listed in the index. (If you are looking for it, it's Section 5.1.1 starting on page 151.) It would be great if it could be included in the index of the next edition. I really hope there will be many editions of this beautiful book.

In general I noticed that the index "feels" short, for instance in comparison with David Watkins' Fundamentals of Matrix Computations.

**Comment by EricDarve** [comment]
Thanks for the feedback. I will make that correction. I do agree that the index needs to be extended.
